docs/VLSI/Building-A-Chip.rst

```diff
@@ -1,6 +1,50 @@
 .. _build-a-chip:
 
 Building A Chip
-==============================
+===============
+
+In this section, we will discuss many of the ASIC-specific transforms and methodologies within Chipyard.
+For the full documentation on how to use the VLSI tool flow, see the `Hammer Documentation <https://hammer-vlsi.readthedocs.io/>`__.
+
+Transforming the RTL
+--------------------
+Building a chip requires specializing the generic verilog emitted by FIRRTL to adhere to the constraints imposed by the technology used for fabrication.
+This includes mapping Chisel memories to available technology macros such as SRAMs, mapping the input and output of your chip to connect to technology IO cells, see :ref:`Barstools`.
+In addition to these required transformations it may also be benefical to transform the RTL to make physical design easier.
```

Comment:
```suggestion
In addition to these required transformations, it may also be beneficial to transform the RTL to make it more amenable to hierarchical physical design.
```

Reply:
Thanks.